lib/authorize.ts

```diff
@@ -1,11 +1,12 @@
-import hostname from './hostname'
+import _hostname from './hostname'
 
 export interface AuthorizeConfig {
   clientId: string;
   uri: string;
+  hostname?: string;
 }
 
-function authorize({ clientId, uri }: AuthorizeConfig): string {
+function authorize({ clientId, uri, hostname = _hostname }: AuthorizeConfig): string {
   return `${hostname}/oauth/authorize?client_id=${clientId}&response_type=code&redirect_uri=${uri}`
```

Comment:
Instead of adding it the main config can we pass it as the second argument like we do for client?

```javascript
authorize({ client_id, uri }, '')
```